### PR TITLE
[FIX] account_invoice_report_grouped_by_picking: filter sections and notes

### PR DESCRIPTION
**Pull request by chienandalu**
Avoid nasty exceptions. In the other hand it would be hard to fit the
concept of sections and notes to the picking groups in which this report
results.

cc @Tecnativa TT29507

ping @sergio-teruel @pedrobaeza

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Please put a note in the ROADMAP about section and lines not being printed with this module.

**Comment by chienandalu** [comment]
Yes. It's already there

File: account_invoice_report_grouped_by_picking/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* As a result of the grouping by pickings we don't support notes and descriptions.
+  Anyway, an invoice with no pickings it should be printed as usual. The problem is
+  that in the current state of the report, we should refactor it to be able to
+  fallback to the regular behavior.
```
Comment (pedrobaeza) [inline account_invoice_report_grouped_by_picking/readme/ROADMAP.rst]:
```suggestion
* As a result of the grouping by pickings, we don't support notes and descriptions.
* Anyways, an invoice with no pickings should be printed as usual. The problem is
  that in the current state of the report, a heavy refactoring is needed to be able to
  fallback to the regular behavior in such case.
```

**Comment by bodedra** [comment]
@pedrobaeza JFI, when I get time, I will work on glue module to support account_invoice_report_grouped_by_picking and account_invoice_report_hide_line module functionality. And @chienandalu will you do forward port for this fix?

File: account_invoice_report_grouped_by_picking/readme/ROADMAP.rst (hunk 1 above)
Comment (chienandalu) [reply account_invoice_report_grouped_by_picking/readme/ROADMAP.rst]: Changed

**Comment by chienandalu** [comment]
@bodedra yes, as soon as it's merged I'll forward it to 13.0

**Review by bodedra — APPROVED** [review]
Code review. :+1:

**Pull request merged**